src/backend/storeManagers/storeManagerCommon/games.ts

```diff
@@ -342,6 +342,11 @@ export async function launchGame(
 
     // Native
     if (isNative) {
+      const isAbsolutePath = path.isAbsolute(executable)
+      const exeOnly = isAbsolutePath
```

Comment:
looks like this is shadowing the outer scope `exeOnly` variable from 
`const { launchArgs, executable: exeOnly } = getExecutableAndArgs(executable)`

Could we rename this variable for readability?

Also could we just reuse the `exeOnly` variable in the outer scope instead of calling 
`getExecutableAndArgs(executable).executable` again?

Reply:
Yeah, it was intentional but I agree its confusing. Will find another way.

Reply:
Moved the fix to the method `getExecutableAndArgs` in `utils` where it should be from the start.